Headscratching.md

```diff
@@ -0,0 +1,26 @@
+Headscratching?
+===============
+
+Here are some things that might help
+
+1. A-Frame ignores components with names containing a capital letter.
```

Comment:
This is only partially true. AFrame uses a deterministic conversion from camel case (myNewComponent) to hyphenated (my-new-component) and vice versa. In HTML you always use the hyphenated version for component names and properties, but you can use either one in javascript.